### PR TITLE
ignore forms that are errors

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?249103#1299502

Form was an errored form that caused it to not have an attachment associated with it (https://github.com/dimagi/commcare-hq/blob/master/corehq/form_processor/models.py#L262)

An errored form shouldn't ever have an effect on cases so we should just skip them

@gcapalbo

### CODE REVIEW

File: corehq/apps/users/tasks.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def tag_forms_as_deleted_rebuild_associated_cases(user_id, domain, form_id_list,
     cases_to_rebuild = set()
 
     for form in FormAccessors(domain).iter_forms(form_id_list):
-        if form.domain != domain:
+        if form.domain != domain or form.is_error:
```
Comment (snopoke) [inline corehq/apps/users/tasks.py]: I wonder if we should exclude all forms except 'normal' ones i.e. errors, duplicates, archived, deprecated etc.

Comment (emord) [inline corehq/apps/users/tasks.py]: Yeah. I had thought about that, but I wasn't sure if anything not normal could change the case. seemed safer to just remove errored forms from this

Comment (emord) [inline corehq/apps/users/tasks.py]: Although if you think it's safe to use `not form.is_normal` i'm happy to change it

**Review by gcapalbo — APPROVED** [review]

File: corehq/apps/users/tasks.py (hunk 1 above)
Comment (snopoke) [inline corehq/apps/users/tasks.py]: Yea, I that would be fine. None of the other's affect cases.

**Review by snopoke — APPROVED** [review]

**Pull request merged**